src/macros.rs

```diff
@@ -86,7 +86,7 @@ macro_rules! core_uri {
 #[macro_export]
 macro_rules! config_uri {
     ($path:expr) => {{
-        format!("{}/{}", $crate::config::base_config_service_url(), $path)
```

Comment:
Why this change?

Reply:
Because we define the slash when using the macro, right now we have two slashes in the url

Reply:
I'd create a different PR for this as it seems outside of the scope for the `AddressEx` changes

Reply:
I created https://github.com/gnosis/safe-client-gateway/issues/489

Reply:
Hahah I would not have minded the fix here xD as this triggers unnecessary forward requests